CMakeLists.txt

```diff
@@ -154,88 +148,63 @@ set(ADEPT_CUDA_SRCS
   src/HostScoring.cu
 )
 
-add_library(CopCore INTERFACE)
-target_include_directories(CopCore
+set(ADEPT_G4_INTEGRATION_SRCS
+  src/AdePTTrackingManager.cc
+  src/AdePTPhysics.cc
+  src/HepEMPhysics.cc
+  src/AdePTGeant4Integration.cpp
+  src/adept_g4_dummy.cu
```

Comment:
The `cuda_rdc_` function are meant to add this when needed.  The fact that it isn't means that the detection mechanism in the `cuda_rdc_` function that the library and/or executable is depending on more than one independent `CUDA rdc` library failed (this could indeed be related to the circular depenendy).